### ISSUE TITLE
Proposing new OSF repository

### ISSUE THREAD

**Issue by josephmje** [carpentries-incubator/SDC-BIDS-dMRI#179]
If we move to using NeuroLibre to host this content, we will be using [Repo2Data](https://github.com/SIMEXP/Repo2Data) to download the pre-processed data from OSF. Repo2Data cannot use the `fetch` command to download specific files and instead using the `clone` command to download all of the files from an OSF repository. 

Can we migrate the existing `ds000221_sub-010006.zip` file to its own OSF repository?

**Comment by kaitj** [comment]
Definitely. I think we can link the two repositories anyways, so we can remove the zip file from the existing and keep it on the separate repo (will have to update existing scripts/instructions/actions).

**Comment by josephmje** [comment]
Awesome! Datalad also now has an [OSF extension](http://docs.datalad.org/projects/osf/en/latest/). Should we consider using this to stay consistent with the IntroMRI lesson and also continue pushing the concepts of dataset version control?

**Comment by kaitj** [comment]
That sounds good to me - I think when this first started there wasn't an extension yet.

Since this is a relatively quick and easy change, I've gone ahead and created a new OSF repo for this and linked it with the old one (data is still present on both for now). The new repo is here: https://osf.io/t7afj/ 

It follows the same structure as before `ds000221_subject/ds000221_sub-010006.zip`

**Comment by jhlegarreta** [comment]
Sounds good for advocating in favor for `datalad`. Please, correct if I'm wrong, but using `datalad` is then independent of the `NeuroLibre` hosting, right?

Also, I'd avoid mixing `datalad` with the binder-related issues in #178: I'd leave transitioning to `datalad` to a separate, dedicated PR. Otherwise, we'd be introducing further inconsistencies.

**Comment by josephmje** [comment]
Using `datalad` would also tie into `NeuroLibre` hosting. If we create a new repository on OSF, we would need to use the datalad OSF extension in order to push the data. And also, this changes what we include in the dataset json file, which contains instructions for downloading the data.
